app/Console/Commands/Node/MakeNodeCommand.php

```diff
@@ -70,6 +69,6 @@ public function handle(): void
         $data['daemon_base'] = $this->option('daemonBase') ?? $this->ask(__('commands.make_node.daemonBase'), '/var/lib/pelican/volumes');
 
         $node = $this->creationService->handle($data);
-        $this->line(__('commands.make_node.succes1') . $data['name'] . __('commands.make_node.succes2')  . $node->id . '.');
+        $this->line(__('commands.make_node.succes1') . $data['name'] . ' ' . __('commands.make_node.succes2')  . $node->id . '.');
```

Comment:
The node name and id should be variables in the translation.
Like `__('commands.make_node.success', ['name' => $data['name'], 'id' => $node->id])`
`Successfully created a new node with the name :name and has an id of :id`

Reply:
oh goodness, can we also fix the "succes2" and "succes1"

Reply:
> oh goodness, can we also fix the "succes2" and "succes1"

That's what I meant :D